src/table/components/TableBodyWrapper.tsx

```diff
@@ -50,9 +50,13 @@ function TableBodyWrapper({
     });
   }, []);
 
+  const totals = (
+    <TableTotals rootElement={rootElement} tableData={tableData} theme={theme} layout={layout} keyboard={keyboard} />
+  );
```

Comment:
I don't know if this is considered less props drilling? still the same properties I pass through, just not as `{...props}`